sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -128,3 +133,16 @@ export class AESTFileUploadToStudentAPIInDTO {
   @ArrayMinSize(1)
```

Comment:
We can remove this class-validator in out DTO.

Reply:
Sorry, I am not following, it is an In DTO? Am I missing something?
![image](https://user-images.githubusercontent.com/61259237/170782699-e3b9d73b-e857-4c7d-9ae6-b83a3f7b6e41.png)

Reply:
My bad this looked like it was added to OutDTO😊

![image](https://user-images.githubusercontent.com/54600590/170789889-9fe6ba51-2416-4a6b-9dc4-789e9dd69c02.png)